components/card/demo/no-padding.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 更灵活的内容展示
+title:
+  zh-CN: 更灵活的内容展示
+  en-US: More flexible content
 ---
 
+## zh-CN
+
 可以调整默认边距，设定宽度。
 
+## en-US
+
+An advanced usage of Card showing the adjustments of default margins like setting the width.
```

Comment:
Thanks for the suggestion, this may be better: 

`An advanced usage of a card, adjusting the width and the default margins.`